namnder/idrottsnamnden/sidebar.md

```diff
@@ -2,7 +2,7 @@
 
 Om du har några sportiga frågor så kan du höra av dig till Sektionsidrottsledaren!
 
-#### Sektionsidrottsledare  
+#### Sektionsidrottsledare
 
-*(Vakant)* </br>
-[idrott@d.kth.se](mailto:idrott@d.kth.se)
+_(Vakant)_ </br>

```

Comment:
```suggestion
Niklas Vatn </br>
```

Reply:
I intentionally didn't update any of the elected officials, but since this PR has been feature-crept to hell and back one might as well...